e2e-tests/version-service/conf/version-service-exact-rs0.yml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: version-service-exact
 spec:
   image: perconalab/percona-server-mongodb-operator:master-mongod4.0
+  initImage: perconalab/percona-server-mongodb-operator:master
```

Comment:
no way, this field calculated from an operator image and different for each feature branch

Reply:
Should be `perconalab/percona-server-mongodb-operator:${branch}` ?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/452/commits/244e13a87dcb41a4e1e2ab4fd58e07913b98a181